components/lists/Posts/PostEditor.tsx

```
@@ -0,0 +1,320 @@
+
+              <div className="prose max-w-none">
+                {editor?.isEmpty ? (
+                  <div className="space-y-2">
+                    <div className="h-4 bg-gray-100 rounded w-full"></div>
+                    <div className="h-4 bg-gray-100 rounded w-5/6"></div>
+                    <div className="h-4 bg-gray-100 rounded w-4/6"></div>
+                  </div>
+                ) : (
+                  <div
+                    dangerouslySetInnerHTML={{
+                      __html: editor?.getHTML() ?? "",
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)